api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -2595,6 +2597,12 @@ type HostedClusterStatus struct {
 	// +patchStrategy=merge
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
+	// Payload represents the CPU architecture type of the HostedCluster.Spec.Release.Image
```

Comment:
Should specify in the godoc which options are valid, users won't know the valid values based on the validations in the CRD unless we document them